chaudloader/src/assets/exedat.rs

```diff
@@ -33,28 +35,53 @@ impl Overlay {
             overlaid_files: std::collections::HashMap::new(),
         }
     }
+    fn normalizePath<'a>(&'a mut self,path: &'a str)->String {
```

Comment:
We don't need the lifetimes here; they can be [elided](https://doc.rust-lang.org/nomicon/lifetime-elision.html).

```rust
fn normalize_path(&mut self, path: str) -> String {
    // ...
}
```

(Note: this probably holds for several other things in the codebase at the moment, like the `read` and `write` in this very file.)